tests/unit/editor/card-lifecycle-test.js

```diff
@@ -93,6 +93,44 @@ test('rendering a mobiledoc for editing calls #unknownCardHandler when it encoun
   editor.render(editorElement);
 });
 
+test('can set initialMode on card to start in non-display mode', (assert) => {
+  let displayCalled, editCalled;
+
+  const name = 'my-card';
+  const cards = [{
+    name,
+    display: {
+      setup() {
+        displayCalled = true;
+      }
+    },
+    edit: {
+      setup() {
+        editCalled = true;
+      }
+    }
+  }];
+
+  const mobiledoc = Helpers.mobiledoc.build(({post}) => {
+    return post();
+  });
+
+  editor = new Editor({ mobiledoc, cards });
+  editor.render(editorElement);
+
+  // must add the card via `run` because the initialMode property would
+  // be lost (if it were set) when serializing the mobiledoc above
+  editor.run(postEditor => {
+    let { builder } = postEditor;
+    let card = builder.createCardSection('my-card');
+    card.initialMode = 'edit';
```

Comment:
yeah, definitely. I was toying with it as a third argument:
`builder.createCardSection(cardName, payload={}, initialMode='display')`

That seems better, right?